tldw_Server_API/app/api/v1/schemas/setup_schemas.py

```
@@ -0,0 +1,194 @@
+        DEFAULT_AUDIO_RESOURCE_PROFILE,
+        min_length=1,
+        description="Selected resource profile within the curated audio bundle.",
+    )
+    pack_name: str | None = Field(
+        None,
+        description="Optional JSON filename to write under the setup-managed audio_packs directory.",
+    )
+
+    @model_validator(mode="before")
+    @classmethod
+    def accept_legacy_pack_path(cls, data: Any) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `data`

(ANN401)